src/pages/UserRegister.vue

```diff
@@ -74,6 +79,7 @@
                         v-model.trim="user.password"
                         type="password"
                         maxlength="80"
+                        :class="errors[0] ? 'border-danger' : ''"
```

Comment:
Hi @PaulSchroeder normally we can use the object in this case
:class="{'border-danger': errors[0] }"